build_run_docker.sh

```diff
@@ -469,6 +469,11 @@ if [ $NUM_MYSQL_NODES -gt 0 ]; then
         template+="$command"
         # template+="$RESOURCES_SNIPPET"
 
+        # mysqld needs this, or will otherwise complain "mbind: Operation not permitted".
```

Comment:
Do you mean when you use valgrind? Since I've never seen that error message before.

Reply:
Hmm, not sure actually. I'll check that, and if so I'll include this in the (future) instrumentation PR instead.

Reply:
Okay cool 👍🏼

Reply:
I can confirm I'm getting this without valgrind, but it seems I need to actually start benchmarks first.